src/Symfony/Component/HttpKernel/Attribute/Cache.php

```diff
@@ -38,27 +38,41 @@ public function __construct(
         public int|string|null $smaxage = null,
 
         /**
-         * Whether the response is public or not.
+         * If true, the contents will be stored in a public cache and serve to all
+         * the next requests.
          */
         public ?bool $public = null,
 
         /**
-         * Whether or not the response must be revalidated.
+         * If true, the response is not served stale by a cache in any circumstance
+         * without first revalidating with the origin.
          */
         public bool $mustRevalidate = false,
 
         /**
          * Additional "Vary:"-headers.
```

Comment:
This might be a bit confusing. "additional" headers ... additional to which ones? Which `Vary` headers do Symfony apply by default?

Also ... when using arrays sometimes is difficult to know exactly the required format:

```php
#[Cache(vary: ['Accept-Language', 'Accept-Encoding'])]

#[Cache(vary: ['Vary' => Accept-Language', 'Vary' => 'Accept-Encoding'])]

#[Cache(vary: ['Vary: Accept-Language', 'Vary: Accept-Encoding'])]

#[Cache(vary: ['Accept-Language' => '*', 'Accept-Encoding' => '*'])]
```

So maybe we could add a very short example.

Reply:
Example added.